dpnp/dpnp_iface_manipulation.py

```diff
@@ -619,6 +620,25 @@ def transpose(x1, axes=None):
     return call_origin(numpy.transpose, x1, axes=axes)
 
 
+def unique(x1, **kwargs):
+    """
+    Find the unique elements of an array.
+
+    For full documentation refer to :obj:`numpy.unique`.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> x = np.array([1, 1, 2, 2, 3, 3])
+    >>> res = np.unique(x)
+    >>> [i for i in res]
```

Comment:
```suggestion
    >>> print(res)
```
Do not create a new object just for printing